webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/reports/StatusReports.scala

```diff
@@ -439,14 +440,14 @@ object ComponentValueStatusReport extends Loggable {
    * them by component *unexpanded* value
    */
   def merge(values: Iterable[ComponentValueStatusReport]): List[ ComponentValueStatusReport] = {
-    val pairs = values.groupBy(_.unexpandedComponentValue).map { case (unexpanded, values) =>
+    val pairs = values.toList /*groupBy(_.unexpandedComponentValue).map { case (unexpanded, values) =>
```

Comment:
this seems dubious

Reply:
I did wanted to no merge them, but i changed my mind and now i want them merged by real value, then by expectedValue